src/gui/speedlimitdialog.h

```diff
@@ -43,17 +47,27 @@ class SpeedLimitDialog : public QDialog
 public:
     explicit SpeedLimitDialog(QWidget *parent);
     ~SpeedLimitDialog();
-    static long askSpeedLimit(QWidget *parent, bool *ok, const QString &title, long defaultVal, long maxVal = 10240000);
+
+public slots:
+    void accept() override;
```

Comment:
Please also change to: `class SpeedLimitDialog final : public QDialog`

Reply:
Done. I'm not sure why you commented on this line though.

Reply:
Github doesn't allow me to place comment there... Also `final` will benefit functions that has `override`.